src/Common/src/Internal/Runtime/EEType.cs

```diff
@@ -435,17 +452,41 @@ internal bool IsGenericTypeDefinition
 #endif
         }
 
+        [StructLayout(LayoutKind.Sequential)]
+        private struct GenericComposition
+        {
+            public readonly ushort Arity;
+
+            private readonly EETypeRef _genericArgument1;
+            public EETypeRef* GenericArguments
```

Comment:
Would relative pointers have any significant benefit for this one?

Reply:
These are shared pretty aggressively. For a hello world on CoreRT, it would save about 8 kB on AMD64.

These leak out of the EEType data structure, so we would need to do something in places where this is used (variant casting and interface dispatch). Given my limited time on this, I would say not worth it for now.